packages/modelviewer.dev/docs/index.html

```diff
@@ -33,7 +33,7 @@
     <div class="home lockup">
       <a href="../" class="sidebar-mv inner-home">
         <div class="icon-button icon-modelviewer-black inner-home"></div>
-        <h1 class="inner-home darken">model-viewer</h1>
+        <h1 class="inner-home darken">&lt;model-viewer&gt;</h1>
```

Comment:
Since this code is shared, would it be possible to load it from somewhere so we can avoid duplication?

Reply:
yeah, I'll refactor that now

Reply:
done!